api/auth/index.ts

```diff
@@ -1,180 +1,179 @@
-import axios, { AxiosResponse } from 'axios'
+import axios, {AxiosResponse} from 'axios'
 import * as express from 'express'
 import * as jwtDecode from 'jwt-decode'
-import { getConfigValue } from '../configuration'
+import * as log4js from 'log4js'
```

Comment:
just a minor change, can we import from our 'log4jui' instead? That sets up the logger.level below and means it's easier to change globally

Reply:
@paddybasi  I made this changes as I saw the latest of Master MO.